src/components/OptionsRow.js

```diff
@@ -1,95 +1,94 @@
-import React, { useState } from 'react';
+import React from 'react';
 import Select from 'react-select';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 
-const plotvars = [
-  { value: 'snw', label: 'Surface Snow Amount' },
+// Some of these might not be relevant anymore
+const plotvarOptions = [
```

Comment:
Would it be better to put all these options in a separate json config file?

Reply:
Probably. I'll open an issue about that and do at a later iteration.